### PR TITLE
[RM Ch09] 9.5.3.1 Pre-Tenant Creation

### PR DESCRIPTION
**Pull request by pgoyal01**
Add content for Pre-Tenant Creation:

1. Tenant Approval -- use, capacity, data centers, ..
1. Tenant Admin, Tenant Member and other Tenant Role approvals for user by role
1. Tenant Images and manifests approvals
1. ....

### CODE REVIEW

**Review by karinesevilla — APPROVED** [review]

**Review by karinesevilla — APPROVED** [review]

File: doc/ref_model/chapters/chapter09.md [hunk 1]
```diff
@@ -214,6 +214,31 @@ All phases described above can be automated using technology specific toolsets a
 <a name="9.5.3.1"></a>
 #### 9.5.3.1. Pre-tenant Creation Requirements
 
+Topics include:
+1. Tenant Approval -- use, capacity, data centers, ..
+    - Validate that the [Tenant's](../../../common/glossary.md#operational-and-administrative-terminology) planned use meets the Operators Cloud Use policies
+    - Validate that the capacity available within the requests cloud site(s) can satisfy the Tenant requested quota for vCPU, RAM, Disk, Network Bandwidth
+    - Validate that the Cloud Infrastructure can meet Tenant's performance requirements (e.g. I/O, latency, jitter, etc)
+    - Validate that the Cloud Infrastructure can meet Tenant's resilience requirements
+1. For VM-based environments:
```
Comment (ulikleber) [inline doc/ref_model/chapters/chapter09.md:223]: We should either add a similar bullet for containerized usage or leave this part for the RAs.

File: doc/ref_model/chapters/chapter09.md [hunk 2]
```diff
@@ -214,6 +214,31 @@ All phases described above can be automated using technology specific toolsets a
...
+    - Verify that the tenant has permissions to use the requested private flavours
+    - Validate that host aggregates are available for specified flavors (public and private)
+    - Verify that the metadata matches for the requested new flavours and host aggregates
+1. Tenant Networks
+    - Verify that the networks requested by the tenant exist
+    - Verify that the security policies are correctly configured to only approved ingress and egress
+1. Tenant Admin, Tenant Member and other Tenant Role approvals for user by role
+    - Add all Tenant Members and configure their assigned roles in the Enterprise Identity and Access management system (e.g., LDAP)
+    - Verify that these roles have been created for the Tenant
+1. Tenant Images and manifests approvals
```
Comment (ulikleber) [inline doc/ref_model/chapters/chapter09.md:235]: The verification of the images doesn't necessarily need to be executed pre-tenant creation. The tenant may later bring additional images etc or update images, which need to be checked as well.
Also this part is VM specific and we need similar text for containers.

**Review by ulikleber — CHANGES_REQUESTED** [review]

File: doc/ref_model/chapters/chapter09.md (hunk 1 above)
Comment (pgoyal01) [reply doc/ref_model/chapters/chapter09.md:223]: @ulikleber The section is for private flavours. Is there something similar for containers?

File: doc/ref_model/chapters/chapter09.md (hunk 2 above)
Comment (pgoyal01) [reply doc/ref_model/chapters/chapter09.md:235]: @ulikleber Thanks. Made a slight edit -- please review.

Comment (ulikleber) [reply doc/ref_model/chapters/chapter09.md:235]: I am still not sure, image verification is to be done **pre-tenant creation**. Or maybe I misunderstood what pre-tenant creation or "tenant image" means.

File: doc/ref_model/chapters/chapter09.md (hunk 1 above)
Comment (pgoyal01) [reply doc/ref_model/chapters/chapter09.md]: @ulikleber Kubernetes doesn't have the concept of flavors -- changed wording to make it independent of VM

**Review by ulikleber — APPROVED** [review]

File: doc/ref_model/chapters/chapter09.md [hunk 3]
```diff
@@ -214,6 +214,31 @@ All phases described above can be automated using technology specific toolsets a
...
+    - Verify and Validate Tenant Images and manifests: virus scan, correct OS version and patch, etc. (Please note that Tenants may also add other images or replace existing images after their environments are created and will also be subjected to image secuirty measures.)
```
Comment (karinesevilla) [inline doc/ref_model/chapters/chapter09.md:236]: @pgoyal01 Good for me
Just a typo:
secuirty -> security

**Review by walterkozlowski — APPROVED** [review]

**Pull request merged**